deploy/helm/sumologic/values.yaml

```diff
@@ -32,6 +32,8 @@ sumologic:
   ## proxy urls
   httpProxy: ""
   httpsProxy: ""
+  ## Exclude kubernetes internal traffic from proxy
+  noProxy: kubernetes.default.svc
```

Comment:
Love this approach it is cleaner.  Con is that if a customer has a different url where they customize the host in the TF kubernetes provider, they have to also update this.  Any way to have one property to control both?

Reply:
Should the cluster endpoint be excluded from the proxy every time?
I can merge both of the config options in helm, but there will be no way to include the kubernetes url then

Reply:
Ahh ok fair point. In that case we can leave it